blockly-ws-search/test/workspace_search_test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Unit tests for WorkspaceSearch.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+const assert = require('assert');
```

Comment:
Don't see the assert in dev dependencies so just curious where it is coming from?

Reply:
I believe its built in to node